init.lua

```diff
@@ -0,0 +1,32 @@
+-- To reload config:
+-- :source %
+-- :so $MYVIMRC
+
+dofile './_plugins.lua'
+dofile './_plugins-settings.lua'
+dofile './_keymaps.lua'
+
+-- Searching.
+vim.opt.ignorecase = true
+vim.opt.smartcase = true
+-- Substituting.
+vim.opt.inccommand = 'nosplit'
+
+-- Misc.
+vim.opt.cursorline = true
+vim.opt.cmdheight = 2
+vim.opt.number = true
+vim.opt.splitbelow = true
+vim.opt.splitright = true
+vim.opt.mouse = 'a'
+vim.opt.listchars = {
+  tab = '→ ',
+  space = '·',
+  nbsp = '␣',
+  trail = '•',
+  eol = '¶',
+  precedes = '«',
+  extends = '»',
+}
+
+-- autocmd BufNewFile,BufRead *.json set syntax=json5
```

Comment:
TODO

Reply:
https://github.com/nanotee/nvim-lua-guide#vimcmd

Reply:
Not necessary anymore https://github.com/kevinoid/vim-jsonc/blob/master/ftdetect/jsonc.vim#L31